docs/gateway/schema.json

```diff
@@ -24,6 +26,20 @@
                         }
                     }
                 }
+            },
+            "data": {
+                "assignments": [
+                    {
+                        "lms": {
+                            "document_url": "https://example.com"
+                        },
+                        "lti": {
+                            "resource_link_id": "23987234",
+                            "resource_link_title": "Assignment name",
+                            "resource_link_description": "Assignment description"
+                        }
```

Comment:
This layout looks a bit weird in isolation, but the thinking in general is something along the lines of:

 * The root looks like the object in `h` speak (in this case there is no equivalent)
 * The `lti` section has things which are expressible in LTI params
 * The `lms` has things which aren't `h` and aren't directly `lti` params either. For example with groups the parent id etc.